graftm/hmmer.py

```diff
@@ -93,9 +89,9 @@ def hmmalign(self, input_path, directions):
             with open(for_file, 'w') as for_aln:
                 logging.debug("Writing forward compliment reads to %s" % for_file)
                 for record in forward:
-                    if record.id and record.seq: # Check that both the sequence and ID fields are there, HMMalign will segfault if not.
-                        for_aln.write('>'+record.id+'\n')
-                        for_aln.write(str(record.seq)+'\n')
+                    if record.id and record.seq:  # Check that both the sequence and ID fields are there, HMMalign will segfault if not.
```

Comment:
Not sure, I might have put this in a while ago, so for now I'll remove and see if I run into any segfaults like it says.